src/Task/FileSize.php

```diff
@@ -76,12 +76,14 @@ public function run(ContextInterface $context)
     {
         $config = $this->getConfiguration();
 
-        if (0 === count($context->getFiles())) {
+        if (0 === $context->getFiles()->count()) {
             return TaskResult::createSkipped($this, $context);
         }
 
         $maxSize = $config['max_size'];
-        $files = $context->getFiles()->size(sprintf('>%s', $maxSize));
+        $files = $context->getFiles()
+            ->size(sprintf('>%s', $maxSize))
+            ->ignoreSymlinks();
```

Comment:
Beter to run the ignoreSymlinks() method before checking the size. That was the actual problem right?